File: app/src/main/java/com/neome/api/meta/base/dto/DefnFieldChipSetDay.java
// neome.ai API - do not change
//

package com.neome.api.meta.base.dto;

import com.neome.api.meta.base.Types.EnumDefnDay;
import com.neome.api.meta.base.Types.MetaIdField;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class DefnFieldChipSetDay extends DefnFieldEditable
{
  @Nullable
  public MetaIdField defaultFieldId;

  @Nullable
  public EnumDefnDay[] defaultValue;

  @Nullable
  public EnumDefnDay[] defaultVar;
}

File: app/src/main/java/com/neome/api/meta/base/dto/DefnFieldChipSetDay.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.dto.DefnFieldEditable
import com.neome.api.meta.base.Types.EnumDefnDay
import com.neome.api.meta.base.Types.MetaIdField

interface DefnFieldChipSetDay : DefnFieldEditable
{
  val defaultFieldId: MetaIdField?
  val defaultValue: Array<EnumDefnDay>?
  val defaultVar: Array<EnumDefnDay>?
}